lib/cli.js

```diff
@@ -100,6 +103,8 @@ exports.run = function () {
 		throw new Error(colors.error(figures.cross) + ' The --require and -r flags are deprecated. Requirements should be configured in package.json - see documentation.');
 	}
 
+	var extensionList = [].concat(cli.flags.extension || 'js');
```

Comment:
I use `extensions` at first. I try to make the name more clear by changed it to `extensionList` .

However, I had followed your suggestion, renamed it back to `extensions`.